### PR TITLE
[analyze] Update pubspec for file fix and add dependabot

### PR DESCRIPTION
**Pull request by CaseyHillers**
This is failing on tip of tree 

See https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8804842406213514465/+/u/analyze/stdout

### CODE REVIEW

**Comment by christopherfujino** [comment]
sorry, i should have fixed this everywhere

**Review by christopherfujino — APPROVED** [review]
LGTM

File: .github/dependabot.yml [hunk 1]
```diff
@@ -28,6 +28,13 @@ updates:
       - "team"
       - "team: infra"
       - "autosubmit"
+  - codesign"
```
Comment (christopherfujino) [inline .github/dependabot.yml:31]: I think you meant:
```suggestion
  - codesign: "pub"
```

Comment (CaseyHillers) [reply .github/dependabot.yml]: Thanks!

**Review by ricardoamador — APPROVED** [review]

**Pull request merged**